### PR TITLE
feat(platform): Adds rating card

### PR DESCRIPTION
**Pull request by Bentlybro**
This adds the rating card, currently i just make it show in every agent page till we have better logic for it + we want to make it show on the block builder when we are looking at some one else's agent from the marketplace, this is to be implemented soon

The Tutorial and ? buttons will also be hidden on the market place pages so ignore them showing for now

### Changes 🏗️
image
image

### CODE REVIEW

File: autogpt_platform/frontend/src/components/agptui/RatingCard.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+} satisfies Meta<typeof RatingCard>;
+
+export default meta;
+type Story = StoryObj<typeof meta>;
+
+export const Default: Story = {
+  args: {
+    agentName: "Test Agent",
+    onSubmit: (rating) => {
+      console.log("Rating submitted:", rating);
```
Comment (majdyz) [inline autogpt_platform/frontend/src/components/agptui/RatingCard.stories.tsx:20]: Is the implementation done yet or it supposed to be a console.debug ?

File: autogpt_platform/frontend/src/components/agptui/RatingCard.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+      console.log("Rating card closed");
+    },
+  },
+};
+
+export const LongAgentName: Story = {
+  args: {
+    agentName: "Very Long Agent Name That Might Need Special Handling",
+    onSubmit: (rating) => {
+      console.log("Rating submitted:", rating);
```
Comment (majdyz) [inline autogpt_platform/frontend/src/components/agptui/RatingCard.stories.tsx:32]: ditto

**Comment by majdyz** [comment]
This looks good, you might need to do `npm run format` or mark the PR as a draft if it's not done yet

File: autogpt_platform/frontend/src/components/agptui/RatingCard.stories.tsx (hunk 1 above)
Comment (Bentlybro) [reply autogpt_platform/frontend/src/components/agptui/RatingCard.stories.tsx:20]: Non of the logic is to be implemented just yet, this is how we have been doing everything, we build the UI first the add the functionality later on, this is for the https://github.com/Significant-Gravitas/AutoGPT/pull/8306 PR

**Pull request merged**